pyproject.toml

```diff
@@ -86,6 +86,8 @@ dev = [
     "stix2==3.0.1",
     "requests==2.31.0",
     "mypy-protobuf==3.6.0",
+    "sarif_om==1.0.4",
+    "jschema_to_python==1.2.3",
```

Comment:
We don't have a good way to manage the requirements for the scripts today. I think you should add these two entries to the script docstring, and maybe wrap their import with a `try/except ImportError` and explain the error to the user.

This `dev` block isn't right, though, since its meant for capa developers and their IDE tooling/code generators.